integration_tests/README.md

```diff
@@ -0,0 +1,38 @@
+# Integration Tests
+
+> [!NOTE]
+> These tests were taken from the reference DAS API implementation, you can find original code in [this repository](https://github.com/metaplex-foundation/digital-asset-rpc-infrastructure).
+
+This Cargo package helps us run multi-package tests in our workspace. This setup is inspired by the tokio integration test setup.
+
+## Setup
+
+First setup a local Postgres database and export the postgres database URL as follows:
+```export DATABASE_TEST_URL=postgres://postgres@localhost/<database_name>```
+
+Also gain access to mainnet RPCs and devnet RPCs and export the URLs as follows. Currently,
+we only use these URLs for reading data and storing it locally. 
+
+```
+export DEVNET_RPC_URL=...
+export MAINNET_RPC_URL=...
+```
+
+Afterwards, you can simply run the following command to run tests:
+```cargo test```
```

Comment:
Don't we need to add `integration_tests` feature here?

Reply:
Oh, sorry, i thought new tests were hidden under this flag. You are right, this change is not needed